docs/content/4.api/1.components/4.content-query.md

```diff
@@ -62,7 +62,7 @@ The `not-found`{lang=ts} slot can be used to display a default content before re
 ```html [pages/about.vue]
 <template>
   <main>
-    <ContentQuery path="/about/authors" :where="{ type: 'csv' }" v-slot="{ data }">
+    <ContentQuery path="/about/authors" :where="{ type: 'csv' }" v-slot="{ authors }">
```

Comment:
We should keep `data` here, and change it in the `v-for` below

```
        <li v-for="author of data" :key="author.name">
          {{ author.name }}
        </li>
```

Reply:
Yes, my mistake. It is fixed now.